website/docs/guides/cli.md

```diff
@@ -0,0 +1,188 @@
+---
+sidebar_position: 3
+---
+
+# CLI
+
+The MCAP command line tool is useful for working with MCAP files.
+
+## Installation
+
+### Release binaries
+
+Download the latest binary for your platform from https://github.com/foxglove/mcap/releases, and mark it executable. For example:
+
+    $ wget https://github.com/foxglove/mcap/releases/latest/download/mcap-linux-amd64 -O mcap
+    $ chmod +x mcap
+
+If required, move the binary onto your path.
+
+### Homebrew
+
+To install using [Homebrew](https://brew.sh) on macOS or Linux, run:
+
+    $ brew install mcap
+
+### Using Go
+
+:::caution
+Installing via 'go install' is not recommended, and is not guaranteed to work.
+:::
```

Comment:
should we just delete this section then?

Reply:
I think it could be helpful having something about how to install from latest main? Honestly I have no idea why this is "not recommended", I moved this text from the existing CLI readme.